src/ui/public/filter_editor/filter_editor.html

```diff
@@ -41,7 +41,7 @@
       <!-- Filter dropdowns -->
       <div
         class="kuiFieldGroup kuiVerticalRhythmSmall kuiFieldGroup--alignTop"
-        ng-show="!filterEditor.showQueryDslEditor()"
```

Comment:
I renamed this because the `show` made it sound imperative. I think prefixing it with `is` makes it clearer that this is checking the state instead of setting it.